lib/sqs/sqs-consumer.js

```diff
@@ -83,6 +83,16 @@ class SqsConsumer extends SqsBase {
     };
   }
 
+  isConsuming() {
+    if(!this._scheduler.isScheduled) return true;
+
+    let now = moment().milliseconds(0),
```

Comment:
Suggestion : Couldn't we just call _getVisibilityTimeout () === 0 ? or refactor to reuse similar logic at both the places ?